invenio_github/api.py

```diff
@@ -147,8 +148,8 @@ def sync(self, sync_hooks=True):
                 'full_name': gh_repo.full_name,
                 'description': gh_repo.description,
             }
-            if sync_hooks:
-                self._sync_hook(gh_repo)
+
+        self._sync_hooks(active_repos.keys(), async=async_hooks)
```

Comment:
Currently the hook sync behaviour goes as:
* Initial GitHub account connection OR Index page access after a full day without sync
  * Triggers an async task that does the hook syncing
* User clicks on "Sync now..." button on index page
  * Triggers a synchronous hook sync through the AJAX request which updates the page on the spot.